lib/jazzy/config.rb

```diff
@@ -220,6 +220,11 @@ def expand_path(path)
         SourceDeclaration::AccessControlLevel.from_human_string(acl)
       end
 
+    config_attr :lint,
+      command_line: '--[no-]lint',
+      description: "Generate linter output.",
+      default: false
```

Comment:
Kind of what I was thinking too — but then wouldn’t this case:

> notify and do not build docs if lint fails

…just be this?

    jazzy --lint || { notify }